src/frontends/CMakeLists.txt

```diff
@@ -25,5 +25,6 @@ if(ENABLE_OV_IR_FRONTEND)
 endif()
 
 if(ENABLE_OV_TF_FRONTEND)
+    add_subdirectory(tensorflow_common)
```

Comment:
Do we need a new macro?
Because tf common can be used without TF FE with TF FE lite.

Reply:
i think i would put it under condition:
```
if (ENABLE_OV_TF_FRONTEND OR ENABLE_OV_TF_LITE_FRONTEND)
    add_subdirectory(tensorflow_common)
```

When TF_LITE would be available in master